spring-petclinic-microservices/README.md

```diff
@@ -18,21 +18,16 @@ You will:
 
 ## What you will need
 
-In order to deploy a Java app to cloud, you need an Azure subscription. If you do not already have
-an Azure subscription, you can activate your
-[MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/)
-or sign up for a
-[free Azure account]((https://azure.microsoft.com/free/)).
-
-In addition, you will need the following:
-
-- [Azure CLI version 2.0.67 or higher](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
-- [Java 8](https://www.azul.com/downloads/azure-only/zulu/?version=java-8-lts&architecture=x86-64-bit&package=jdk)
-- [Maven](https://maven.apache.org/download.cgi)
+- [An Azure subscription](https://azure.microsoft.com/free/)
```

Comment:
It failed for me to run the samples with docker, can you check and run it locally?